example-signing-ipld.mjs

```diff
@@ -0,0 +1,88 @@
+// JWT & utilities
+import {
+  ES256KSigner,
+  createJWS,
+  verifyJWS
+} from 'did-jwt'
```

Comment:
We've been discussing renaming this library to `did-jose` since it does much more than just JWTs :)